syncer/checkpoint.go

```diff
@@ -15,6 +15,7 @@ package syncer
 
 import (
 	"fmt"
+	"github.com/pingcap/tidb-tools/pkg/dbutil"
```

Comment:
re-order it.

Reply:
https://github.com/pingcap/dm/pull/840/commits/54db3d8b21ab19943448e5a1de07ced2a533f56b